nshlib/nsh_parse.c

```diff
@@ -603,24 +603,31 @@ static int nsh_execute(FAR struct nsh_vtbl_s *vtbl,
     {
       FAR char *sh_argv[4];
       FAR char *sh_cmd = "sh";
-      int i;
+      char cmd_str[CONFIG_NSH_LINELEN];
```

Comment:
shall we consider the stack usage?

Reply:
> shall we consider the stack usage?

Yes, and I had a check, the default value is 64/80 bytes, is this ok?
https://github.com/apache/nuttx-apps/blob/1c7a7f7529475b0d535e2088a9c4e1532c487156/nshlib/Kconfig#L128C1-L134C17

Reply:
```suggestion
      char sh_arg2[CONFIG_NSH_LINELEN];
```

Reply:
> char cmd_str[CONFIG_NSH_LINELEN];
>       char sh_arg2[CONFIG_NSH_LINELEN];

done